cmake/projects/Avahi/hunter.cmake

```diff
@@ -20,10 +21,11 @@ hunter_add_version(
 )
 
 hunter_cmake_args(
-    Avahi 
+    Avahi
     CMAKE_ARGS
-      BUILD_SHARED_LIBS=ON  
+      BUILD_SHARED_LIBS=ON
 )
 
+hunter_configuration_types(Avahi CONFIGURATION_TYPES Release)
 hunter_pick_scheme(DEFAULT url_sha1_avahi_autotools)
 hunter_download(PACKAGE_NAME Avahi)
```

Comment:
> Oh, I see. So adding PACKAGE_INTERNAL_DEPS_ID "1" should be enough, correct?

yes

> I ask to make sure because there are quite many files to update

Yes, it's true. Also there are a lot of packages to test.